<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBook-Air.local" failures="1" tests="9" name="com.createIncident.ios.CreateIncident_114" time="1466.971" errors="1" timestamp="2023-08-20T17:34:50 IST" skipped="7">
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="configAm" time="0.781">
    <failure message="Test Case Fail" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test Case Fail
at org.testng.Assert.fail(Assert.java:97)
at Base.BaseTest.configAm(BaseTest.java:204)
at jdk.internal.reflect.GeneratedMethodAccessor66.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:658)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:642)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- configAm -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts01incident" time="1466.190">
    <error message="Expected condition failed: waiting for visibility of Located by By.xpath: //XCUIElementTypeButton[@name=&quot;Next&quot;] (tried for 180 second(s) with 500 milliseconds interval)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;MacBook-Air.local&#039;, ip: &#039;fe80:0:0:0:1c84:a0af:f027:5e0c%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;12.6.8&#039;, java.version: &#039;11.0.18&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {automationName: XCUITest, browserName: , bundleId: com.threedi.OhioNFERsQA, databaseEnabled: false, deviceName: IPhone13, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: IOS, platformVersion: 16.5.1, safariIgnoreFraudWarning: true, takesScreenshot: true, udid: 00008110-001A71600E28201E, useNewWDA: false, webStorageEnabled: false}
Session ID: 55b24343-0f63-46a2-bf5a-28882677194a" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of Located by By.xpath: //XCUIElementTypeButton[@name="Next"] (tried for 180 second(s) with 500 milliseconds interval)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'MacBook-Air.local', ip: 'fe80:0:0:0:1c84:a0af:f027:5e0c%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '12.6.8', java.version: '11.0.18'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {automationName: XCUITest, browserName: , bundleId: com.threedi.OhioNFERsQA, databaseEnabled: false, deviceName: IPhone13, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: IOS, platformVersion: 16.5.1, safariIgnoreFraudWarning: true, takesScreenshot: true, udid: 00008110-001A71600E28201E, useNewWDA: false, webStorageEnabled: false}
Session ID: 55b24343-0f63-46a2-bf5a-28882677194a
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at utils.WebDriverUtility.waitForElemnetToBeVisible(WebDriverUtility.java:80)
at Base.BasePage.click(BasePage.java:97)
at pom.createincidentios.Basicinfo.nextbtn(Basicinfo.java:122)
at com.createIncident.ios.CreateIncident_114.ts01incident(CreateIncident_114.java:122)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ts01incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts02incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts02incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts03incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts03incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts04incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts04incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts05incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts05incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts06incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts06incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts07incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts07incident -->
  <testcase classname="com.createIncident.ios.CreateIncident_114" name="ts08incident" time="0.000">
    <skipped/>
  </testcase> <!-- ts08incident -->
</testsuite> <!-- com.createIncident.ios.CreateIncident_114 -->
